man/tpm2_create.1.md

```diff
@@ -90,10 +97,19 @@ These options for creating the tpm entity:
 
 # EXAMPLES
 
+Create an object whose parent is provided via parent.ctx:
+```
+tpm2_create -C parent.ctx -u obj.pub obj.priv
+```
+
+Create an object and seal data to it:
+```
+tpm2_create -C parent.ctx  -K def456 -G keyedhash -I seal.dat -u obj.pub -r obj.priv
+```
+
+Create an rsa2048 object and load it into the TPM:
 ```
-tpm2_create -C 0x81010001 -P abc123 -K def456 -g sha256 -G keyedhash-I data.File
-tpm2_create -C file:parent.context -P abc123 -p def456 -g sha256 -G keyedhash -I data.File
-tpm2_create -C 0x81010001 -P 123abc -K 456def -X -g sha256 -G keyedhash -I data.File
+tpm2_create -C primary.ctx -G rsa2048 -u obj.pub -r obj.priv -c obj.ctx
```

Comment:
Should be -o obj.ctx now

Reply:
Good catch... these always seem to miss being updated